src/KNoTThing.cpp

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2016, CESAR.
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the BSD license. See the LICENSE file for details.
+ *
+ */
+
+#include <stdint.h>
+
+#include "KNoTThing.h"
+
+KNoTThing::KNoTThing()
+{
+	
+}
+
+KNoTThing::~KNoTThing()
+{
+	knot_thing_exit();
+}
+
+int KNoTThing::init()
+{
+	return knot_thing_init();
+}
```

Comment:
Does it make sense to remove this method moving the function call to KNoTThing's contructor?